pages/docs/couple-your-code/couple-your-code-gradient-data.md

```diff
@@ -69,6 +90,42 @@ while (not simulationDone()){ // time loop
 /* ... */
 ```
 
+</div>
+<div role="tabpanel" class="tab-pane" id="python-2" markdown="1">
+
+```python
+precice = precice.Participant("FluidSolver", "precice-config.xml", rank, size)
+
+mesh_dim = precice.get_mesh_dimensions("FluidMesh")
+data_dim = precice.get_data_dimensions("FluidMesh", "Stress")
+
+vertex_ids = precice.set_mesh_vertices("FluidMesh", coords)
+
+stress = np.zeros(vertex_size*data_dim)
+stress_gradient = np.zeros(vertex_size*data_dim*mesh_dim)
```

Comment:
Those need to be multi-dimensional, no?

Reply:
I was thinking the same, but in cpp it was declared as ```vector<double>``` instead of  ```vector<vector<double>>``` . 
![image](https://github.com/user-attachments/assets/3e269690-55b6-474e-b650-5e71e2fbfb2d)

Reply:
Yes, the C++ and the Python API differ here. In C++ it's a one-dimensional array, in Python, it is multi-dimensional.